MetadataProcessor.Shared/Utility/nanoCLR_DataType.cs

```diff
@@ -52,6 +52,25 @@ public enum nanoCLR_DataType : byte
         DATATYPE_SZARRAY, // Shortcut for single dimension zero lower bound array SZARRAY <type>
         DATATYPE_BYREF, // BYREF <type>
 
+        /// <summary>
```

Comment:
These can't be added here sorry... otherwise there will be a mismatch between the value for the enum here and at the native code running on the MCU.